src/Form/input-classifiers.test.js

```diff
@@ -139,8 +139,24 @@ describe('Input Classifiers', () => {
         })
     })
 })
-let testResults = []
 
+const getMismatchedValue = (score) => {
```

Comment:
Had to tweak how we compare in the test suite so we can account for the new `variant` without breaking all existing tests. I still back-filled about 100 of them 💪 .